src/net/atlas/db.rs

```diff
@@ -387,7 +387,7 @@ impl AtlasDB {
         content_hash: &Hash160,
     ) -> Result<Option<Attachment>, db_error> {
         let hex_content_hash = to_hex(&content_hash.0[..]);
-        let qry = "SELECT content, hash FROM attachments WHERE hash = ?1 AND was_instantiated = 0"
+        let qry = "SELECT content, hash FROM attachments WHERE hash = ?1 AND was_instantiated = 1"
```

Comment:
There haven't been any attachments propagated to the mainnet network yet, right?  i.e. We're certain this change isn't going to break anything?

Reply:
The BNS endpoints + registrar not being live at this point, the only interactions that happened with the contract were internal tests (https://explorer.stacks.co/txid/SP000000000000000000002Q6VF78.bns?chain=mainnet)